log/log.go

```diff
@@ -72,7 +72,7 @@ func init() {
 	       return
 	   }
 	*/
-	logrus.SetFormatter(&logrus.TextFormatter{})
+	logrus.SetFormatter(&logrus.JSONFormatter{})
```

Comment:
what's benefit changing TextFormatter to JSONFormatter?

Reply:
just for test, I will change it back